classes/email/driver/mailgun.php

```diff
@@ -20,14 +20,24 @@ protected function _send()
 
 		$message = $this->build_message();
 
-		$mg = \Mailgun\Mailgun::create($this->config['mailgun']['key']);
-		
+		$config_mailgun = $this->config['mailgun'];
+
+		$endpoint = isset($config_mailgun['endpoint']) ? $config_mailgun['endpoint'] : null;
+
+		if (empty($endpoint)) {
```

Comment:
Coding standard: bracket on the next line please

Reply:
Sure thing. The source seems to be a mix of different standards, so we weren't sure what to do, and I ended up with some mix of the two. Will update and re-submit.

Reply:
Would you like copyright dates updated?

From

``2010 - 2019 Fuel Development Team``

to 

``2010 - 2020 Fuel Development Team``

Reply:
> The source seems to be a mix of different standards

That's not good, I'll go through it.

> Would you like copyright dates updated?

No, it's ok, I've got a script for it, I'll change it to 2021 next month...

Reply:
> > The source seems to be a mix of different standards
> 
> That's not good, I'll go through it.
> 
Well, I don't mean to overstate the mix.

In addition to making the requested change, I've changed the foreach brackets in the mailgun driver source to conform, and have updated the docs as requested. Will push changes later when able.